### ISSUE TITLE
Screenreaders do not read the new date when it is changed in Scheduler

### ISSUE THREAD

**Issue by nelito987** [telerik/kendo-ui-core#6550]
When the dates for the current view are changed in Scheduler, the screenreaders do not read the new date. 

### Reproduction of the problem
1. Go to [Keyboard Navigation Demo](https://demos.telerik.com/kendo-ui/scheduler/selection)
1. Go to the arrow buttons in the toolbar and click 'Next' to change the date 
1. Date changes to "Sunday, June 16, 2013 - ..."

### Current behavior
The screenreaders do not inform the user that the date has been changed.

### Expected/desired behavior
The user should be informed that the date has been changed. 

### Environment

* **Kendo UI version:** 2021.2.616
* **Browser:** [all ]
